staging/src/k8s.io/client-go/tools/cache/listwatch.go

```diff
@@ -102,9 +99,8 @@ func NewFilteredListWatchFromClient(c Getter, resource string, namespace string,
 
 // List a set of apiserver resources
 func (lw *ListWatch) List(options metav1.ListOptions) (runtime.Object, error) {
-	if !lw.DisableChunking {
-		return pager.New(pager.SimplePageFunc(lw.ListFunc)).List(context.TODO(), options)
-	}
+	// ListWatch is used in Reflector, which already supports pagination.
```

Comment:
I extracted this one into: https://github.com/kubernetes/kubernetes/pull/86790
As discussed with @liggitt offline, I believe that we want to do this independently of how/if we will proceed with this PR.